docs/src/guide/periodic_problems.jl

```diff
@@ -0,0 +1,321 @@
+#     H v_{kj} = ε_{kj} v_{kj}
+# ```
+# satisfy a factorisation
+# ```math
+#     v_{kj}(x) = e^{i k⋅x} ψ_{kj}(x)
+# ```
+# into a plane wave ``e^{i k⋅x}`` and a lattice-periodic function
+# ```math
+#    T_{na} ψ_{kj}(x) = ψ_{kj}(x - na) = ψ_{kj}(x) \quad ∀ n ∈ \mathbb{Z}.
+# ```
+# In this ``j`` is a labelling integer index and ``k`` is a real number,
+# whose details will be clarified in the next section.
+# Functions ``v_{kj}`` satisfying this factorisation are also known as
+# **Bloch functions** or **Bloch states**.
+#
+# Applying ``2H = -Δ = - \frac{d^2}{d x^2}`` to such a Bloch wave yields
+# ```math
+# \begin{aligned}
+# -\Delta \left(e^{i k⋅x} ψ_{kj}(x)\right)
+#    &= k^2 e^{ik⋅x} ψ_{kj}(x) - 2ik e^{ik⋅x} (∇ψ_{kj}(x)) - e^{ik⋅x} (Δψ_{kj}(x)) \\
+#    &= e^{ik⋅x} (k^2 - 2ik ∇ - Δ) ) ψ_{kj}(x) \\
+#    &= e^{i k⋅x} (-i∇ + k)^2 ψ_{kj}(x) \\
+#    &= e^{i k⋅x} 2H_k ψ_{kj}(x),
+# \end{aligned}
+# ```
+# where we used
+# ```math
+#     H_k = \frac12 (-i∇ + k)^2.
+# ```
+# The action of this operator on a function ``ψ_{kj}`` is given by
+# ```math
+#     H_k ψ_{kj} = e^{-i k⋅x} H e^{i k⋅x} ψ_{kj},
+# ```
+# which in particular implies that
+# ```math
+#    H_k ψ_{kj} = ε_{kj} ψ_{kj} \quad ⇔ \quad H (e^{i k⋅x} ψ_{kj}) = ε_{kj} (e^{i k⋅x} ψ_{kj}).
+# ```
+# To seek the eigenpairs of ``H`` we may thus equivalently
+# find the eigenpairs of *all* ``H_k``.
+#
+# A more detailed mathematical analysis shows that the transformation from ``H``
+# to the set of all ``H_k`` for a suitable set of values for ``k`` (detailes below)
+# is actually a unitary transformation, the so-called **Bloch transform**.
+# This transform brings the Hamiltonian into the symmetry-adapted basis for
+# translational symmetry, which are exactly the Bloch functions.
+# Similar to the case of choosing a symmetry-adapted basis for other kinds of symmetries
+# (like the point group symmetry in molecules), the Bloch transform also makes
+# the Hamiltonian ``H`` block-diagonal[^1]:
+# ```math
+#     T_B H T_B^{-1} ⟶ \left( \begin{array}{cccc} H_1&&&0 \\ &H_2\\&&H_3\\0&&&\ddots \end{array} \right)
+# ```
+# with each block ``H_k`` taking care of one value ``k``.
+# This block-diagonal structure under the basis of Bloch functions lets us
+# completly describe the spectrum of ``H`` by looking only at the the spectrum
+# of all ``H_k`` blocks.
+#
+# [^1]: Notice that block-diagonal is a bit an abuse of terms here, since the Hamiltonian
+#       is not a matrix but an operator and the number of blocks is essentially infinite.
+#       The mathematically precise term is that the Bloch transform reveals the fibers
+#       of the Hamiltonian.
+#
+# ## The Brillouin zone
+#
+# The big mystery in the discussion so far is the parameter ``k`` of the Hamiltonian blocks.
+#
+# - As discussed ``k`` is essentially a real number. It turns out, however, that some of
+#   these ``k∈\mathbb{R}`` give rise to operators related by unitary transformations
+#   (again due to translational symmetry).
```

Comment:
Just explain it from the ansatz psi(x) = e^ikx u(x): k and k+2pi/a give the same class of functions